pkg/client/networks.go

```diff
@@ -0,0 +1,163 @@
+package client
+
+import (
+	"github.com/ethereum/go-ethereum/common"
+)
+
+// Network represents a Tableland network.
+type Network string
+
+const (
+	// Testnet is the Tableland testnet network.
+	Testnet Network = "https://testnet.tableland.network"
+	// Staging is the Tableland staging network.
+	Staging Network = "https://staging.tableland.network"
+	// Localhost is the Tableland local network.
+	Localhost Network = "http://localhost:8080"
+)
```

Comment:
This isn't sustainable, but it's the way it works now in the single validator world. Will revisit when we need to.

Reply:
Good point. I added it for consistency with the JS SDK. Also maybe there will be a reason for a user to use it in a support situation with us. Will leave it for now for these reasons, but open to further changes here in the future if we decide they are needed.

Reply:
The SDK includes all chains, but we don't advertise the staging ones on READMEs / docs. The CLI has a `tableland chains` command that list all supported chains, but staging ones are filtered out.